src/components/dataset/summary/row.vue

```diff
@@ -14,9 +14,12 @@ except according to the terms contained in the LICENSE file.
     <div class="row">
       <div class="col-xs-6 dataset-name-wrap">
         <div class="dataset-name text-overflow-ellipsis" v-tooltip.text>
-          <router-link :to="datasetPath(projectId, dataset.name)" v-tooltip.text>
+          <router-link v-if="!dataset.isNew" :to="datasetPath(projectId, dataset.name)" v-tooltip.text>
             {{ dataset.name }}
           </router-link>
+          <template v-else>
```

Comment:
Instead of `<template>`, how about `<span v-tooltip.text>` in case the dataset name is very long?

Reply:
the outer div `.dataset-name` seems to take care of this already without changes!

<img width="539" alt="Screen Shot 2023-09-12 at 5 37 11 PM" src="https://github.com/getodk/central-frontend/assets/76205/50630e67-aae5-4b18-9704-ff587ef80946">